src/frontend/NavigationStacks/WithModalsStack.tsx

```diff
@@ -11,7 +11,7 @@ const MainStack = createSwitchNavigator(
     Onboarding: OnboardingStack,
   },
   {
-    initialRouteName: "App",
+    initialRouteName: "Onboarding",
```

Comment:
this is an intentional change right?

Reply:
Is it? haha. I think we should do this for the APK we build, but i just did this for testing and forgot to switch it back

Reply:
ah ok then maybe let's just revert it to `App` and adjust for when we create the build